packages/rum-react/test/e2e/with-router/general.e2e-spec.js

```diff
@@ -79,14 +79,10 @@ describe('General usecase with react-router', function () {
       'expected manual component to be rendered'
     )
 
-    const { sendEvents } = waitForApmServerCalls(0, 2)
+    const { sendEvents } = waitForApmServerCalls(0, 1)
     const { transactions } = sendEvents
 
-    const pageLoadTransaction = transactions[0]
```

Comment:
This is not necessary this step triggers a soft navigation. (without resetting the mock as we are doing right now we would not be able to doing that, so another benefit)